### PR TITLE
replace composite primary key with uuid

### PR DESCRIPTION
**Pull request by windhustler**

### CODE REVIEW

File: src/main/kotlin/com/ampnet/headlesscmsservice/persistence/model/Mail.kt [hunk 1]
```diff
@@ -16,7 +17,8 @@ import javax.persistence.Table
 class Mail(
 
     @Id
-    val id: Int,
+    @Column
+    val id: UUID,
```
Comment (githinho) [inline src/main/kotlin/com/ampnet/headlesscmsservice/persistence/model/Mail.kt]:
```suggestion
    val uuid: UUID,
```

**Review by githinho — CHANGES_REQUESTED** [review]
Rename all mail `id` to `uuid`.

File: src/main/kotlin/com/ampnet/headlesscmsservice/service/pojo/MailResponse.kt [hunk 2]
```diff
@@ -3,9 +3,10 @@ package com.ampnet.headlesscmsservice.service.pojo
 import com.ampnet.headlesscmsservice.enums.Lang
 import com.ampnet.headlesscmsservice.enums.MailType
 import com.ampnet.headlesscmsservice.persistence.model.Mail
+import java.util.UUID
 
 data class MailResponse(
-    val id: Int?,
+    val id: UUID?,
```
Comment (githinho) [inline src/main/kotlin/com/ampnet/headlesscmsservice/service/pojo/MailResponse.kt]: Why not `uuid` to be more concise?

Comment (windhustler) [reply src/main/kotlin/com/ampnet/headlesscmsservice/service/pojo/MailResponse.kt]: My bad, I missed it!

**Review by githinho — APPROVED** [review]

**Pull request merged**